client/src/Components/relatedItems/relatedCarousel/relatedCard/cardComponents/comparison.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Comparison({ trigger, handlesComparePopup, data, displayProduct }) {
+  return (
+    trigger ?
+    <div className="comparison-popup"> {data.name} {displayProduct.name}
```

Comment:
Separating jsx expressions causes multiple text elements to render. If this is intended, disregard, otherwise try using string literals

Reply:
sorry, this was intended to be a test to see if both data were rendering